apps/spruce/src/components/SpruceForm/Container.tsx

```diff
@@ -16,7 +17,11 @@ export const SpruceFormContainer: React.FC<ContainerProps> = ({
   title,
 }) => (
   <div>
-    {title && <SettingsCardTitle id={id}>{title}</SettingsCardTitle>}
+    {title && (
+      <a href={`#${transformTitleToId(title)}`} id={transformTitleToId(title)}>
+        <SettingsCardTitle id={id}>{title}</SettingsCardTitle>
+      </a>
+    )}
```

Comment:
I looks like the headers for the array fields don't get the anchor link because they are part of a different object. Could we file a ticket for it? The list would be:
* Parsley Filters on Views & Filters page
* Metadata Links on Plugins page
* Patch Aliases & Patch Trigger Aliases on Patch Aliases page
* Subscriptions on Notifications page